tensorflow/python/keras/layers/pooling.py

```diff
@@ -446,7 +446,11 @@ def __init__(self,
 @keras_export('keras.layers.AveragePooling2D', 'keras.layers.AvgPool2D')
 class AveragePooling2D(Pooling2D):
   """Average pooling operation for spatial data.
-
+  
+  >>> x = tf.keras.Input(shape=(3, 3, 1), name='x')
+  >>> average_pooling_layer = tf.keras.layers.AveragePooling2D(pool_size = (1,1),strides = (1,1),padding = "same")(x)
+  <tf.Tensor 'average_pooling2d_3/Identity:0' shape=(None, 3, 3, 1) dtype=float32>
```

Comment:
Thanks for your contribution , can you please remove chevron(>>>) symbol  and limit the line length to 80 characters.
Here is the link for [contribution guidelines.](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#google-python-style-guide)